### ISSUE TITLE
Accessibility Bug: "Take me There" Button does not work in "Platform Pricing" page  #287

### ISSUE THREAD

**Issue by suganyapravin** [topcoder-platform/Training-Challenges#287]
Steps to reproduce:

Go to https://www.topcoder.com/
Select "Pricing"
Click on "Start a Program Conversation" under Enterprise Programs
Click on "Take Me There" button towards the bottom of the page

Actual Result:
Nothing happens on clicking "Take Me There" button

Expected Result:
Should take me to the top of the page on clicking the button.

Screenshots:

![Screen Shot 2021-10-21 at 2 32 41 PM](https://user-images.githubusercontent.com/59360647/138337115-0c87f5fb-2699-45d8-9945-175e67453fee.png)
![Screen Shot 2021-10-21 at 2 33 03 PM](https://user-images.githubusercontent.com/59360647/138337120-321a8553-ab5a-4627-89b4-0854ca8b4514.png)

Device/OS/Browser Information: Laptop/Mac book/Safari

**Comment by sdgun** [comment]
This page is changed now and "Take Me There" button is removed, so this issue is not present now.
